sdk/communication/azure-communication-common/src/main/java/com/azure/communication/common/MicrosoftTeamsUserIdentifier.java

```diff
@@ -107,6 +89,29 @@ public CommunicationCloudEnvironment getCloudEnvironment() {
         return cloudEnvironment;
     }
 
+    /**
+     * Get full id of the identifier. This id is optional.
+     * @return full id of the identifier
+     */
+    public String getRawId() {
+        return rawId;
+    }
+
+    @Override
+    public String getId() {
```

Comment:
`getId` should not exist. Is it a method in the base class? If so, it should be removed. `getId` only should exist in CommunicationUserIdentifier